web_src/css/base.css

```diff
@@ -1249,7 +1249,7 @@ img.ui.avatar,
 }
 
 .ui .text.truncate {
-  overflow-x: hidden;
+  overflow: hidden;
```

Comment:
This brings again the danger of cutting off letters, but overall I'm in favor of cutting off instead of presenting a scrollbar. Cut-off letters should be fixed with a appropriate `line-height`.

See https://github.com/go-gitea/gitea/issues/25597 and linked PRs.